src/util/StringUtils.h

```diff
@@ -458,12 +466,9 @@ inline vector<string> splitWsWithEscape(const string& orig, const char left,
 }
 
 // _____________________________________________________________________________
-vector<string> splitAny(const string& orig, const char* seps) {
-  return splitAny(orig, string(seps));
-}
-
-// _____________________________________________________________________________
-vector<string> splitAny(const string& orig, const string& seps) {
+template <class T>
+vector<string> splitAny(const string& orig, const T& seperators) {
+  std::string_view seps(seperators);
```

Comment:
maybe comment that this ensures us to have a size() method when we come from string  literals etc. I was a bit confused at first why this is done.